parastell.py

```diff
@@ -699,11 +645,22 @@ def parastell(
     wall_s = build['wall_s']
     # Extract radial build
     radial_build = build['radial_build']
-
     # Extract array dimensions
     n_phi = len(phi_list)
     n_theta = len(theta_list)
 
+    # Compute toroidal extent of build in degrees
+    tor_ext = phi_list[-1] - phi_list[0]
```

Comment:
per previous comment, I think it makes sense to convert this to radians ASAP and everything else in radians internally